front/defenses/service.js

```diff
@@ -1,15 +1,15 @@
-import selector from '@/components/front/selector'
+import selector from '@/front/selector'
 
 export default function makeDefensesServices(axios) {
   return {
     fetchPage(params) {
       const paramList = ['page', 'query', 'course', 'status']
-      const payload = selector(params, paramList, { page: 1 })
+      const payload = { params: selector(params, paramList, { page: 1 }) }
```

Comment:
[Não é um payload](https://stackoverflow.com/questions/23118249/whats-the-difference-between-request-payload-vs-form-data-as-seen-in-chrome).

`options`